### ISSUE TITLE
ISettingsProvder is not loaded from adapter.settingsprovider.dll with --Platform:x64

### ISSUE THREAD

**Issue by karanjitsingh** [Microsoft/vstest#1673]
I have two dlls `testadapter.dll` and `testadapter.settingsprovider.dll` with settings name as `JSTest`

I get this when I run `vstest.console.exe` with `/Platform:x64`

```
Microsoft (R) Test Execution Command Line Tool Version 15.7.0
Copyright (c) Microsoft Corporation.  All rights reserved.

Starting test execution, please wait...
An exception occurred while invoking executor 'executor://jstestadapter/v1': Settings Provider named 'JSTest' was not found.  The settings can not be loaded.

Test Run Failed.
```

If `ISettingsProvider` implementation is put in the `testadapter.dll` itself instead of having two dlls, the `/Platform:x64` works however it throws the same error for x86 i.e without the platform switch.

TL;DR
`ISettingsProvider` needs a `testadapter.settingsprovder.dll` to work with x86, breaks with x64
`ISettingsProvider` needs to be in the `testadapter.dll` to work with x64, but breaks with x86
